lib/common/strings.c

```diff
@@ -600,3 +600,26 @@ crm_strdup_printf(char const *format, ...)
     va_end(ap);
     return string;
 }
+
+bool
```

Comment:
Let's use the new standard return codes, pcmk_rc_ok instead of true, and a reasonable errno instead of false.